core-plugins/widgets/Projection-transform.json

```diff
@@ -51,9 +51,39 @@
             "key-placeholder": "Field Name",
             "value-placeholder": "New Field Name"
           }
+        },
+        {
+          "plugin-function": {
+          "method": "POST",
+          "widget": "outputSchema",
+          "label": "Get Schema Value",
+          "required-fields": [],
+          "plugin-method": "getSchema"
+          }
         }
       ]
     }
   ],
-  "outputs": []
+  "outputs": [
```

Comment:
I don't think you need this, there doesn't need to be a default schema.

Reply:
Update the output part.